CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact agate module, newest changes on the top.
 
+## unreleased
+
+- Updated dependencies including `immer` to version `^10.0.2`.
+- Updated `babel-loader` version to `^9.1.2` for sampler.
+- Updated the minimum version of Node to `14.15.0`for sampler.
```

Comment:
i don't think sampler updates need to be added to the changelog of the agate library